includes/wp-admin/table/class-followers.php

```diff
@@ -118,7 +118,20 @@ public function process_action() {
 					}
 				}
 				break;
+			case 'follow':
+				$redirect_to = \remove_query_arg( array( 'follower', 'followers' ), $redirect_to );
 
+				if ( isset( $_GET['follower'], $_GET['_wpnonce'] ) ) {
+					$follower = \absint( $_GET['follower'] );
+					$nonce    = \sanitize_text_field( \wp_unslash( $_GET['_wpnonce'] ) );
+
+					if ( \wp_verify_nonce( $nonce, 'follow-follower_' . $follower ) ) {
+						Following::follow( $follower, $this->user_id );
+
+						\add_settings_error( 'activitypub', 'follower_followed', \__( 'User followed successfully.', 'activitypub' ), 'success' );
```

Comment:
```suggestion
						\add_settings_error( 'activitypub', 'followed', \__( 'Account followed.', 'activitypub' ), 'success' );

```
Could we re-use the same message as in the following table?